source/contracts/factories/MarketFactory.sol

```diff
@@ -13,7 +13,7 @@ contract MarketFactory {
         Delegator _delegator = new Delegator(_controller, "Market");
         _market = IMarket(_delegator);
         IReputationToken _reputationToken = _universe.getReputationToken();
-        _reputationToken.transfer(_market, _reputationToken.balanceOf(this));
+        require(_reputationToken.transfer(_market, _reputationToken.balanceOf(this)));
```

Comment:
Per comments previously, this is wasteful.  Why do we wrap this one function call in a `require` while every other cross contract function call is not wrapped in a require statement?  What makes this one special?  This goes against our existing coding standards and implies to the user that this contract call is somehow special/different from the others even though it is not.  It also is a waste of gas.

Reply:
(same comment for many other additions in this PR)

Reply:
I tend to agree. The argument being made is one of adhering to standards without special consideration since that can expose vulnerabilities from absentnt-mindely not using a requires in a case where an external token is used.